cli/pkg/stack_mgr/stack_service.go

```diff
@@ -210,7 +209,7 @@ func (s *StackService) removeFromStacklist(ctx context.Context, stackName string
 	return s.writeStacklist(ctx, stackNamesList)
 }
 
-func (s *StackService) Add(ctx context.Context, stackName string, dryRun bool, options ...workspace_repo.TFERunOption) (*Stack, error) {
+func (s *StackService) Add(ctx context.Context, stackName string, dryRun bool, options ...workspacerepo.TFERunOption) (*Stack, error) {
```

Comment:
if we are adding run options to all these methods, there is a DryRun run option, so we can probably replace the dryRun argument in all of them with a DryRun TFERunOption

Reply:
Let's do it in a separate PR -- this change is too intrusive.